### DISCUSSION TITLE
SSO with Azure AD should be free for the Team tier

### DISCUSSION THREAD

**Discussion by dcuccia** [community/community#28691] [Other Feature Feedback, Questions, & Ideas]
All of our employees are already Microsoft 365 E5 members. I think it's a crazy leap from $4/mo to $21/mo per user for the privilege to use MS's own identity provider. I agree with https://sso.tax/ - SSO is a basic requirement of any company over 5 users, and should not be lumped with premium services. We just set up Salesforce SSO with Azure AD, and it was a breeze and no extra charge.

**Comment by techretailo** [comment]
so what alternative for this ? as we were thinking to move to Gitlab free tier to GitHub Team account but GitHub team account does not have SSO

**Reply by dcuccia** [reply]
@techretailo none that I know of
